charts/argo/templates/server-cluster-role.yaml

```diff
@@ -1,6 +1,10 @@
 {{- if .Values.server.enabled }}
 apiVersion: rbac.authorization.k8s.io/v1
+{{- if .Values.singleNamespace }}
+kind: Role
+{{ else }}
 kind: ClusterRole
+{{- end }}
 metadata:
   name: {{ .Release.Name }}-{{ .Values.server.name}}-cluster-role
```

Comment:
The name of this and other roles are now slightly confusing, we should either use `-role` or `-cluster-role` based on what it is or drop the suffix altogether.

However this could be a breaking change if others are re-using the existing roles for other purposes...

Reply:
Agree 100%. Changed all the names to the correct one depending on the `singleNamespace` flag.